q2_quality_control/tests/test_quality_control.py

```diff
@@ -562,7 +569,8 @@ def test_evaluate_composition_mockrobiota_metadata_map(self):
         res = _evaluate_composition(
             self.exp, self.obs, depth=7, palette='Set1',
             plot_tar=True, plot_tdr=True, plot_r_value=True,
-            plot_r_squared=True, plot_observed_features=True,
+            plot_r_squared=True, plot_bray_curtis=False,
```

Comment:
Can you add one text that explicitly plots bray curtis? Just to make sure everything succeeds.

Reply:
This test now plots jaccard and bray_curtis. Thanks for checking! Not sure why I did not do that before...